models/ape_store/base/ape_store_base_trades.sql

```diff
@@ -0,0 +1,104 @@
+{{ config(
+    alias = 'trades',
+    schema = 'ape_store_base',
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['blockchain', 'tx_hash', 'tx_index'],
+    post_hook = '{{ expose_spells(
+                        blockchains = \'["base"]\',
+                        spell_type = "project",
+                        spell_name = "ape_store",
+                        contributors = \'["whale_hunter"]\') }}'
```

Comment:
as noted above, let's remove for now. i'd rather play it safe, as we don't have a process to remove from data explorer at the moment.